test/test_privateuseone_python_backend.py

```diff
@@ -0,0 +1,156 @@
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+import numpy as np
+import torch
+from torch.overrides import TorchFunctionMode
+from torch.utils._pytree import tree_map
+import torch._C
+import torch.utils._mode_utils as mode_utils
+from torch.utils.backend_registration import setup_privateuseone_for_python_backend
+
+import numpy as np
+
+
+setup_privateuseone_for_python_backend('npy')
+
+aten = torch.ops.aten
+
+# NOTE: From https://github.com/albanD/subclass_zoo/blob/main/new_device.py
+# but using torch.library instead of `__torch_dispatch__`
+class MyDeviceTensor(torch.Tensor):
+
+    @staticmethod
+    def __new__(cls, size, dtype, raw_data=None, requires_grad=False):
+        # Use a meta Tensor here to be used as the wrapper
+        res = torch._C.create_empty_tensor(size, dtype)
+        res.raw_data = raw_data
+        return res
+
+    def __init__(self, size, dtype, raw_data=None, requires_grad=False):
```

Comment:
`__init__` will not be called because `__new__` don`t return instance of `MyDeviceTensor` or the subclass of it.